structure_tests/structure_test.go

```diff
@@ -34,59 +35,59 @@ func TestRunCommand(t *testing.T) {
 		stdout := outbuf.String()
 		stderr := errbuf.String()
 
-		for _, errStr := range tt.ExpectedError {
+		for _, errStr := range tt.GetExpectedError() {
```

Comment:
One other way to think about this might be to have a separate TestSchemaX function for each schema version, where that Test is a no-op unless there are tests defined with it's schema.